.travis.yml

```diff
@@ -75,10 +75,10 @@ jobs:
     - rvm: ruby-head
       env: RGV=master
       stage: test
-    # 1.x mode (we want to keep stuff passing in 1.x mode for now)
-    - rvm: 2.6.1
-      env: RGV=v2.7.8 BUNDLER_SPEC_SUB_VERSION=1.98
-      stage: test
+    # 3.x mode
+    # - rvm: 2.5.3
+    #   stage: test
+    #   env: RGV=v3.0.2 BUNDLER_SPEC_SUB_VERSION=3.0.0
```

Comment:
@colby-swandale I think we need to get this passing, otherwise we'll be no longer making sure that _all_ new features developed in master are working. Something I noticed that's also incorrect is that we should set the development version to 3.0.0.dev, because otherwise _none_ of the new features will be actually tested.

What failures are you getting and why can't they be fixed?